### DISCUSSION TITLE
it will be nice if we can parse the lspci and the pci.ids

### DISCUSSION THREAD

**Discussion by braindevices** [kellyjonbrazil/jc#280] [Ideas]
the lspci has some kind of machine readable output but not very easy to parse when try to use verbose output. There were attempts to add `-J` option to output json format, but those patches had never got merged.

pci.ids are also very useful especially the class database at the end `# List of known device classes, subclasses and programming interfaces`. If we can parse it and generate a `class number: class name` dictionary, it will be very helpful.

**Comment by kellyjonbrazil** [comment]
Hi there - thank you for the parser suggestion! Go ahead and open a [GitHub Issue](https://github.com/kellyjonbrazil/jc/issues) and we'll track it there. It sounds like supporting the `-mm` and possibly `-mmv` options would be desired?

**Reply by braindevices** [reply]
@kellyjonbrazil yeap. If we can support `-mmv` it will be very useful too.
